inc/post-content-filters.php

```diff
@@ -37,6 +37,12 @@ public static function row_attributes( $attributes, $row ){
 		if( ! empty( $row['style'] ) ) {
 			$attributes[ 'data-style' ] = json_encode( $row['style'] );
 		}
+		if( ! empty( $row['ratio'] ) ) {
+			$attributes[ 'data-ratio' ] = intval( $row['ratio'] );
```

Comment:
Wait, missed this. Can't ratio also be a float?

Reply:
Ah, good catch! :+1: